### PR TITLE
#51 - removing ecs

### PR DESCRIPTION
**Pull request by krzysztofrewak**
I get rid of the ECS in the project. It depends solely on PHP-CS-Fixer now.

#### Upgrading guide from 0.x
With version 1.x we removed `symplify/easy-coding-standard` dependency in the project. The checklist for updating old projects is as follows:
- [ ] update the main dependency `blumilksoftware/codestyle` to version `^1.0` in `composer.json` file
- [ ] run `composer update blumilksoftware/codestyle -W`
- [ ] rename `ecs.php` to `codestyle.php`
- [ ] update scripts in `composer.json` file
- [ ] update scripts in Github Actions
- [ ] the constructor of `Blumilk\Codestyle\Config` lost two parameters: `$sets` and `$skipped`; all manipulations of rules should be done on base `$rules` list
- [ ] `Blumilk\Codestyle\Configuration\Defaults\LaravelPaths` returns a default Laravel 9 directory schema; for Laravel 8 additional parameter `LaravelPaths::LARAVEL_8_PATHS` should be added

### CODE REVIEW

File: tests/unit/LaravelPathsConfigurationTest.php [hunk 1]
```diff
@@ -14,18 +14,29 @@ public function testLaravelPathsConfiguration(): void
         $config = new Config(paths: $paths);
 
         $this->assertSame(
-            ["app", "config", "database", "resources/lang", "routes", "tests"],
+            ["app", "bootstrap/app.php", "config", "database", "lang", "public/index.php", "routes", "tests"],
+            $config->options()["paths"],
+        );
+    }
+
+    public function testPreviousLaravelVersionPathsConfiguration(): void
```
Comment (kamilpiech97) [inline tests/unit/LaravelPathsConfigurationTest.php:22]: Maybe we will tell which version is it? When `Laravel` has version `10`, will this test need to be updated?

**Review by kamilpiech97 — APPROVED** [review]

File: tests/unit/LaravelPathsConfigurationTest.php (hunk 1 above)
Comment (krzysztofrewak) [reply tests/unit/LaravelPathsConfigurationTest.php:22]: Yes, but this will be a breaking change.

File: src/Config.php [hunk 2]
```diff
@@ -4,73 +4,79 @@
...
-                if ($configuration) {
-                    $service->call("configure", [$configuration]);
-                }
-            }
-        };
+        $config = new PhpCsFixerConfig("Blumilk codestyle standard");
+        return $config->setFinder($finder)
+            ->setUsingCache(false)
+            ->registerCustomFixers(new PhpCsFixerCustomFixers())
+            ->registerCustomFixers([new DoubleQuoteFixer()])
```
Comment (mtracz) [inline src/Config.php:49]: Can we define method which will return custom fixers to be registered? And why the first fixes is object and the second one is an array?

**Review by mtracz — CHANGES_REQUESTED** [review]

File: src/Config.php (hunk 2 above)
Comment (krzysztofrewak) [reply src/Config.php:49]: First is a object, because it has `IteratorAggregate` interface. `registerCustomFixers()` takes everything that is iterable.

Comment (krzysztofrewak) [reply src/Config.php]: Method extracted.

**Review by mtracz — APPROVED** [review]

**Pull request merged**